sangbok/main.xml

```diff
@@ -4592,4 +4592,54 @@
 			hur innerligt jag älskar dig!
 		</p>
 	</song>
+	
+	<song name="I Californien"
+		  category="Esoterica">
```

Comment:
Had time to sit down and look at this just now, turns out there are a number of versions for the melody, but I'm pretty certain I managed to find the correct one.